_includes/alerts.html

```diff
@@ -1,8 +1,7 @@
 <div class='tease'>
   <p>
   <a>Conference season is here! Join us at FOSDEM &amp; Config Management Camp, 2nd - 6th February 2019</a>
-  <a>We'd also love to hear your Foreman story at Configuration Management Camp!</a>
-    <br/>
-    <a href='https://cfp.cfgmgmtcamp.be'>Submit your talk proposal here! (closes 1st Dec)</a>
+  <br/>
+  <a href='https://da.gd/tfm_survey'>The 2019 Foreman Community Survey is now open! Click here to participate!</a>
```

Comment:
I prefer full URLs if they're links anyway. URL shorteners are for humans :)

Reply:
It's about 1 billion characters of a Google Forms URL and I hate it :P but I'll put it in if you insist...